google/cloud/forseti/services/explain/service.py

```diff
@@ -47,6 +52,18 @@ def _get_handle(self, context):
             metadata_dict[key] = value
         return metadata_dict[self.HANDLE_KEY]
 
+    def _determine_is_supported(self):
+        """Determine whether Explainer is supported for clients to use.
+
+        Returns:
+            bool: True if Explainer module can be used.
+        """
+        root_resource_id = (
+            self.explainer.config.inventory_config.root_resource_id)
+        if 'organizations' in root_resource_id:
```

Comment:
Is this the only thing that needs to be checked? What about if dwd is not configured?